datadog/fwprovider/resource_datadog_integration_gcp_sts.go

```diff
@@ -52,6 +53,11 @@ func (r *integrationGcpStsResource) Schema(_ context.Context, _ resource.SchemaR
 	response.Schema = schema.Schema{
 		Description: "Provides a Datadog Integration GCP Sts resource. This can be used to create and manage Datadog - Google Cloud Platform integration.",
 		Attributes: map[string]schema.Attribute{
+			"account_tags": schema.SetAttribute{
+				Optional:    true,
+				Description: "Tags to be associated with GCP metrics and service checks from your account.",
+				ElementType: types.StringType,
```

Comment:
Hmmm, I wonder why this is `types.StringType` even though above it's `types.Set`. The REST API does have `account_tags` as an optional `array` of strings 🤔

Reply:
If I'm not wrong (not a Go expert) this is the type of the elements in the set (schema.SetAttribute)
This is inspired from what has been done for host_filters which is has the same type

Reply:
It's indicating what each element in the set are. (ie: This is a set of strings)

[Reference](https://developer.hashicorp.com/terraform/plugin/framework/handling-data/attributes/set)

Reply:
Thanks! Resolving then.